### ISSUE TITLE
Improve UI

### ISSUE THREAD

**Issue by ramyashreeshetty** [TheLastProject/Droidtoberfest#3]
some changes in UI can be made.
choosing the project with a description provided below it can make it user friendly.

**Comment by TheLastProject** [comment]
Yeah would indeed be good. I want to first focus on getting my own Hacktoberfest PRs in but I'll keep running this repo next year too.

I don't think I'm allowed to make this repo part of Hacktoberfest as it's similar to a "list repo", so people fixing it won't count as Hacktoberfest contribution.

**Comment by ramyashreeshetty** [comment]
@TheLastProject  no problem i understand

**Comment by TheLastProject** [comment]
I still want to fix this at some point, even if it won't count for Hacktoberfest :P
